stuff/cvarlist.md

```diff
@@ -81,7 +81,7 @@ Audio:
 * **ogg_enable**: Enable Ogg/Vorbis music playback.
 
 * **ogg_ignoretrack0**: Normally Quake II disabled the background music
-  if a major objective has been archived by setting the music track to 0.  
```

Comment:
Those double spaces at the line endings have a meaning: They make markdown insert at <br>, i.e. new line but not new paragraph. please leave them in.
(The single spaces at end of lines are indeed superfluous)

Maybe your editor stripped spaces at line ends automatically; Generally I recommend using git gui (or similar) to create commits, and only commit lines you actively changed (=> leave whitespace changes from your editor in other parts of the file(s) out of the commit).

Reply:
> Those double spaces at the line endings have a meaning...

Good to know. I restored whitespaces in dc3da331f5dc081513c20c60b40b2c06936a4cfc.